iradix_test.go

```diff
@@ -1532,26 +1532,29 @@ func TestLenTxn(t *testing.T) {
 	}
 }
 
-func TestIterateLowerBound(t *testing.T) {
-	fixedLenKeys := []string{
-		"00000",
-		"00001",
-		"00004",
-		"00010",
-		"00020",
-		"20020",
-	}
-
-	mixedLenKeys := []string{
-		"a1",
-		"abc",
-		"barbazboo",
-		"foo",
-		"found",
-		"zap",
-		"zip",
-	}
+// these should be defined in order
```

Comment:
My preference would be to see this data copied to the new test method rather than shared, but that's probably because I've had to maintain code before where the pattern of shared test data had over time resulted in a mess that no one understood enough to feel safe modifying.  I don't see that as very likely here, although I do think the new test method would be a little bit more readable if it had the data defined inline.